.github/workflows/plugins-test.yaml

```diff
@@ -300,6 +300,8 @@ jobs:
         run: ./mvnw --batch-mode -f test/plugin/pom.xml clean package -DskipTests docker:build -DBUILD_NO=local >/dev/null
       - name: Run spring 4.1.x-4.2.x (20)
         run: bash test/plugin/run.sh spring-4.1.x-scenario
+      - name: Run resttemplate 1.5.0.RELEASE-1.5.22.RELEASE (23)
```

Comment:
Name is not right. Notice, There is another group name. You should change that too.

Reply:
https://github.com/apache/skywalking/blob/47c84deea7f534f4cbb37d3b089b4b2d0ca4a4c7/.github/workflows/plugins-test.yaml#L275